src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2688,6 +2694,7 @@ public override ICustomTypeDescriptor GetTypeDescriptor([DynamicallyAccessedMemb
 
                 if (instance == null)
                 {
+                    // TODO-NULLABLE: If this returned new EmptyCustomTypeDescriptor() then base class's return value could likely be not null
```

Comment:
We'd used "TODO-NULLABLE" when we were doing massive amounts of annotation and we needed a way to track categories of issues.  I think we're past that point :)  I suggest removing this comment and just opening an issue about this particular case.

Reply:
I've opened:
- https://github.com/dotnet/runtime/issues/55176
- https://github.com/dotnet/runtime/issues/55177
- https://github.com/dotnet/runtime/issues/55178

and removed TODO-NULLABLE